web/js/layers/layers.model.js

```diff
@@ -1,17 +1,16 @@
-/**
- * @module wv.layers
- */
-var wv = wv || {};
-wv.layers = wv.layers || {};
+import each from 'lodash/each';
+import eachRight from 'lodash/eachRight';
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import findIndex from 'lodash/findIndex';
+import isUndefined from 'lodash/isUndefined';
+import util from '../util/util';
 
-/**
- * @class wv.layers.model
- */
-wv.layers.model = wv.layers.model || function (models, config) {
+export function layersModel(models, config) {
```

Comment:
When you are just exporting one function like this one. I would suggest exporting it as the default `export default function (models, config) { }`

Reply:
I was thinking if we leave out `default`, we have to reference the name of the function we're importing, which makes it easier to search the codebase for that function and know that it will have the same name. When we use `default`, we're naming variables whatever we want, which leaves room for error. We had this discussion in Slack and came to that conclusion; https://nasa-gibs.slack.com/archives/C6X6QDJ0J/p1509637754000232

Reply:
👍 I will update my branch